script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_1.txt

```diff
@@ -62,13 +62,13 @@ I'm still alive. It's a serious wound, but it's not fatal.
 I don't know where I am, but in a few more minutes, the paralysis in my limbs will subside and I'll be able to move again.
 When that happens, I can fight back.
 <Page16>
-This time I'll be on my guard. I'll get a hold of one of those two and get some information out of―――
+This time I'll be on my guard. I'll get ahold of one of those two, and this time I'll be the one asking―――
```

Comment:
I think "get a hold" is also correct and more used?
Also, "this time" is repeated twice, maybe change unless it's on purpose.